### DISCUSSION TITLE
Getting 401 Unauthorized

### DISCUSSION THREAD

**Discussion by kstmostofa** [awcodes/filament-curator#549] [Q&A]
![curator error](https://github.com/user-attachments/assets/a8daed0d-a4a2-4622-a8b5-6c2f4c91988a)

upload is working fine on my local. but in production server there I am using docker getting this error

**Comment by awcodes** [comment]
Sounds like a permissions issue on your server. If it's working fine on local then I'm not sure this is an issue with the plugin.

**Comment by kstmostofa** [comment]
Using filament default fileupload file is uploading fine in server.
Issue are occurring in this only

**Reply by awcodes** [reply]
I hear you, but if it's working locally and not in production then something is different between the two environments.

**Comment by kstmostofa** [comment]
In production i am using docker to deploy
And in local i am using laravel herd

**Reply by lyrihkaesa** [reply]
check `app/Http/Middleware/TrustProxies.php` 
```php
<?php

namespace App\Http\Middleware;

use Illuminate\Http\Middleware\TrustProxies as Middleware;
use Illuminate\Http\Request;

class TrustProxies extends Middleware
{
    /**
     * The trusted proxies for this application.
     *
     * @var array|string
     */
    protected $proxies = '*'; // change it
    // or
    protected $proxies = ['100.100.100.100']; // ip nginx proxy.
    /**
     * The headers that should be used to detect proxies.
     *
     * @var int
     */
    protected $headers =
        Request::HEADER_X_FORWARDED_FOR |
        Request::HEADER_X_FORWARDED_HOST |
        Request::HEADER_X_FORWARDED_PORT |
        Request::HEADER_X_FORWARDED_PROTO |
        Request::HEADER_X_FORWARDED_AWS_ELB;
}
```
